build.gradle.kts

```diff
@@ -29,10 +29,12 @@ repositories {
 
 // Dependencies are managed with Gradle version catalog - read more: https://docs.gradle.org/current/userguide/platforms.html#sub:version-catalog
 dependencies {
+    buf("build.buf:buf:${libs.versions.buf.get()}:${osdetector.classifier}@exe")
```

Comment:
Trick that I used in protovalidate-java to download the buf CLI instead of building it.